app/views/content.js

```diff
@@ -13,21 +13,27 @@ const Content = Mn.LayoutView.extend({
         tracks: '.tracks',
     },
 
-    modelEvents: {
-        'change:currentPlaylist': 'switchPlaylist'
+    onBeforeShow() {
+        this.showChildView('header', new HeaderView({ model: this.model }));
+        this.showChildView('tracks', new TracksView({
+            collection: application.allTracks.get('tracks')
+        }));
+        this.setClass(application.appState.get('currentPlaylist'));
     },
 
-    onBeforeShow () {
-        this.showChildView('header', new HeaderView({ model: this.model }));
-        this.switchPlaylist(null, this.model.get('currentPlaylist'));
+    initialize() {
+         this.listenTo(application.appState,
+            'change:currentPlaylist',
+            (appState, currentPlaylist) => {
+                this.setClass(currentPlaylist);
+            }
+        );
     },
 
-    // only the router change the current playlist
-    switchPlaylist (appState, currentPlaylist) {
-        this.showChildView('tracks', new TracksView({
-            collection: currentPlaylist.get('tracks')
-        }));
-    }
+    setClass(currentPlaylist) {
+        let type = currentPlaylist.get('tracks').type;
+        this.$el.children('.tracks').removeClass().addClass('tracks ' + type);
```

Comment:
If what you want is just to add the `type` as class on the `.tracks` children, no need to remove classes before. Simply do a 
```js
this.$el.children('.tracks').addClass(type);
```

(see https://api.jquery.com/addClass/)